### ISSUE TITLE
When enabled, Windows app breaks DNS resolution in WSL 2 VMs.

### ISSUE THREAD

**Issue by restfuladi** [nextdns/windows#8]
Disabling the app resolves the issue. Domains are not in blocklist, show as resolved in the Web GUI, and dig seems to work.

```$ dig archive.ubuntu.com

; <<>> DiG 9.16.1-Ubuntu <<>> archive.ubuntu.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 45943
;; flags: qr rd ad; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 0
;; WARNING: recursion requested but not available

;; QUESTION SECTION:
;archive.ubuntu.com.            IN      A

;; ANSWER SECTION:
archive.ubuntu.com.     0       IN      A       91.189.88.142
archive.ubuntu.com.     0       IN      A       91.189.88.152

;; Query time: 0 msec
;; SERVER: 172.25.16.1#53(172.25.16.1)
;; WHEN: Wed Jul 21 10:38:49 EDT 2021
;; MSG SIZE  rcvd: 86

$ sudo apt update
Err:1 http://archive.ubuntu.com/ubuntu focal InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:2 http://security.ubuntu.com/ubuntu focal-security InRelease
  Temporary failure resolving 'security.ubuntu.com'
Hit:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease
Reading package lists... Done
Building dependency tree
Reading state information... Done
9 packages can be upgraded. Run 'apt list --upgradable' to see them.
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.

$ dig security.ubuntu.com

; <<>> DiG 9.16.1-Ubuntu <<>> security.ubuntu.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 35132
;; flags: qr rd ad; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 0
;; WARNING: recursion requested but not available

;; QUESTION SECTION:
;security.ubuntu.com.           IN      A

;; ANSWER SECTION:
security.ubuntu.com.    0       IN      A       91.189.91.39
security.ubuntu.com.    0       IN      A       91.189.91.38

;; Query time: 0 msec
;; SERVER: 172.25.16.1#53(172.25.16.1)
;; WHEN: Wed Jul 21 10:39:31 EDT 2021
;; MSG SIZE  rcvd: 88

$ sudo apt update
Err:1 http://archive.ubuntu.com/ubuntu focal InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:2 http://security.ubuntu.com/ubuntu focal-security InRelease
  Temporary failure resolving 'security.ubuntu.com'
Hit:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease
Reading package lists... Done
Building dependency tree
Reading state information... Done
9 packages can be upgraded. Run 'apt list --upgradable' to see them.
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.```

**Comment by rs** [comment]
The resolver needs to be changed to 192.0.2.42 in WSL to fix this.

**Comment by restfuladi** [comment]
This worked, thank you!

**Comment by rs** [comment]
Will be fixed with v3.0.0
